.github/workflows/bbw_build_container.yml

```diff
@@ -24,6 +24,12 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - dockerfile: centos.Dockerfile
```

Comment:
As AlmaLinux is binary compatible with RHEL why do we need a container? We can just use the same binaries and with a VM run an install  test.

Reply:
These builders are missing and prevent a transition from the old buildbot to the new one. From my side, we can drop them, but @grooverdan can you please talk with the others and come up with a decision?